.github/CODEOWNERS

```diff
@@ -0,0 +1,231 @@
+/helmfile.d/values/kube-prometheus-stack-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-state-metrics-extra-resource-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kured.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/letsencrypt.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/metrics-server.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/minio.yaml.gotmpl @elastisys/goto-pipeline-qa-release
+/helmfile.d/values/node-local-dns.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/openstack-monitoring.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/prometheus-alerts-sc.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/prometheus-blackbox-exporter-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/prometheus-opensearch-exporter.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/prometheus-user-alerts-wc.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/s3-exporter.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/sc-servicemonitor.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/tekton.gotmpl @elastisys/goto-continuous-delivery
+/helmfile.d/values/user-alertmanager.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/user-rbac.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/velero-* @elastisys/goto-security
+/helmfile.d/values/wc-servicemonitor.yaml.gotmpl @elastisys/goto-monitoring-stack
+
+#
+# /helmfile.d/
+#
+
+/helmfile.d/README.md @elastisys/goto-scripts
+/helmfile.d/state.yaml @elastisys/goto-scripts
+
+#
+# images
+#
+
+/images/backup-postgres/ @elastisys/goto-postgresql
+/images/calico-accountant/ @elastisys/goto-kubernetes
+/images/elasticsearch-curator/ @elastisys/goto-logging-stack
+/images/log-manager/ @elastisys/goto-logging-stack
+/images/rclone-sync/ @elastisys/goto-security
+
+#
+# Skipped files in images
+# curl-jq/
+#
+
+/migration/ @elastisys/goto-pipeline-qa-release
+
+/pipeline/ @elastisys/goto-pipeline-qa-release
+
+/release/ @elastisys/goto-pipeline-qa-release
+
+/restore/harbor/ @elastisys/goto-container-registry
+
+/restore/rclone/ @elastisys/goto-security
+
+/roles/ @elastisys/goto-scripts
+
+#
+# scripts
+#
+
+/scripts/ @elastisys/goto-scripts
+/scripts/azure/ @elastisys/goto-azure
+/scripts/bom/ @elastisys/goto-pipeline-qa-release
+/scripts/grafana-dashboards @elastisys/goto-monitoring-stack
+/scripts/migration/ @elastisys/goto-pipeline-qa-release
+/scripts/report/ @elastisys/goto-pipeline-qa-release
+/scripts/sbom/ @elastisys/goto-pipeline-qa-release
+
+
+/tests/ @elastisys/goto-pipeline-qa-release
+
+#
+# /
+#
+
+/.editorconfig @elastisys/goto-pipeline-qa-release
+/.pre-commit-config.yaml @elastisys/goto-pipeline-qa-release
+/README.md @elastisys/goto-public-docs
+/REQUIREMENTS @elastisys/goto-scripts
+/SECURITY.md @elastisys/goto-security
+
+#
+# Skipped files in /
+/.gitignore
+# DEVELOPMENT.md
```

Comment:
This really should have an owner since it's filled with a lot of stuff.
My candidates are:
@elastisys/goto-scripts since it touches on a lot of the local cluster setup which this team currently owns as part of /scripts + @aarnq has written a lot of it
@elastisys/goto-public-docs because it's docs

Reply:
I'll add goto-scripts for this one.